### PR TITLE
Meeting Minutes for 4/21 and 4/28 plus format fixes

### PR DESCRIPTION
**Pull request by sandersms**
Signed-off-by: Mark Sanders <Mark_S_Sanders@dell.com>

### CODE REVIEW

File: doc/minutes/OPI API Minutes-42822.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+- Use Case Initial Discussion
+
+Notes:
+
+- Use Case Discussion
+  - Virtualized Network (OVS) with Security Groups (VxLAN / Geneve) topology
+    - OVN or Kubernetes type of infrastructure
+    - Virtual Routing
+  - Start with a software based solution first with 2 servers, KVM, etc.
+  - Base Use case agreed to on two physical servers (example redhat blog)
```
Comment (glimchb) [inline doc/minutes/OPI API Minutes-42822.md:20]: https://www.redhat.com/en/blog/optimizing-server-utilization-datacenters-offloading-network-functions-nvidia-bluefield-2-dpus

File: doc/minutes/OPI API Minutes-42822.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    - Can we standardize the device data models?
+    - Abstract the interface with port/device information
+  - Develop initial diagram for review in Draw.io <https://app.diagrams.net/>
+  - Use to begin concept creation to identify any gaps in OpenConfig (ex. Virtual Switching gaps)
+
+Actions:
+
+- Dan Daly (Intel) to provide initial view of Use case Diagram to the github site.
+- Fabrizio provided RHEL files for OCP concept and config
+- Intel, Marvell to provide device data model views for Dataplane/Consumption Interface
```
Comment (glimchb) [inline doc/minutes/OPI API Minutes-42822.md:35]: @mkalderon agreed to upload the Marvel's scripts as well

**Review by glimchb — APPROVED** [review]
lgtm

File: doc/minutes/OPI API Minutes-42822.md (hunk 2 above)
Comment (mkalderon) [reply doc/minutes/OPI API Minutes-42822.md:35]: will upload a readme instead of scripts to make it more clear (basically generated from the scripts)

**Pull request merged**